Something changed for RunDotNetCore artifacts output

Previously `BuildArtifacts\temp\_PublishedLibraries\{project}` contained framework moniker for multitargeted builds
```
net46
   dll/pdb/xml/etc.
netstandard1.6
   dll/pdb/xml/etc.
```
Now we just see `dll/pdb/xml/etc.` for one framework directly in that folder with no framework monikers.

Pinned to `0.3.0-unstable0342` and now the builds work again,.

Don't know if it's a lead or not but saw these warnings in the build log
```
C:/projects/cake-kudu-client/tools/Cake.Recipe.0.3.0-unstable0354/Content/build.cake(241,12): warning CS0618: 'CustomProjectParserResult.OutputPath' is obsolete: 'Use OutputPaths instead for multi-targeting support'
C:/projects/cake-kudu-client/tools/Cake.Recipe.0.3.0-unstable0354/Content/build.cake(243,44): warning CS0618: 'CustomProjectParserResult.OutputPath' is obsolete: 'Use OutputPaths instead for multi-targeting support'
C:/projects/cake-kudu-client/tools/Cake.Recipe.0.3.0-unstable0354/Content/build.cake(271,36): warning CS0618: 'CustomProjectParserResult.OutputPath' is obsolete: 'Use OutputPaths instead for multi-targeting support'
C:/projects/cake-kudu-client/tools/Cake.Recipe.0.3.0-unstable0354/Content/build.cake(328,32): warning CS0618: 'CustomProjectParserResult.OutputPath' is obsolete: 'Use OutputPaths instead for multi-targeting support'
C:/projects/cake-kudu-client/tools/Cake.Recipe.0.3.0-unstable0354/Content/build.cake(337,32): warning CS0618: 'CustomProjectParserResult.OutputPath' is obsolete: 'Use OutputPaths instead for multi-targeting support'
C:/projects/cake-kudu-client/tools/Cake.Recipe.0.3.0-unstable0354/Content/build.cake(345,32): warning CS0618: 'CustomProjectParserResult.OutputPath' is obsolete: 'Use OutputPaths instead for multi-targeting support'
C:/projects/cake-kudu-client/tools/Cake.Recipe.0.3.0-unstable0354/Content/build.cake(353,32): warning CS0618: 'CustomProjectParserResult.OutputPath' is obsolete: 'Use OutputPaths instead for multi-targeting support'
C:/projects/cake-kudu-client/tools/Cake.Recipe.0.3.0-unstable0354/Content/build.cake(361,32): warning CS0618: 'CustomProjectParserResult.OutputPath' is obsolete: 'Use OutputPaths instead for multi-targeting support'
```

--- a/Cake.Recipe/Content/build.cake
+++ b/Cake.Recipe/Content/build.cake
@@ -356,9 +356,26 @@ public void CopyBuildOutput()
         else
         {
             Information("Project has an output type of library: {0}", parsedProject.RootNameSpace);
-            var outputFolder = BuildParameters.Paths.Directories.PublishedLibraries.Combine(parsedProject.RootNameSpace);
-            EnsureDirectoryExists(outputFolder);
-            CopyFiles(GetFiles(parsedProject.OutputPath.FullPath + "/**/*"), outputFolder, true);
+
+            // If .NET SDK project, copy for each output path
+            // Otherwise just copy
+            if(parsedProject.IsVS2017ProjectFormat)
+            {
+                foreach(var outputPath in parsedProject.OutputPaths)
+                {
+                    var outputFolder = BuildParameters.Paths.Directories.PublishedLibraries.Combine(parsedProject.RootNameSpace).Combine(outputPath.GetDirectoryName());
+                    EnsureDirectoryExists(outputFolder);
+                    Information(outputPath);
+                    CopyFiles(GetFiles(outputPath + "/**/*"), outputFolder, true);
+                }
+            }
+            else
+            {
+                var outputFolder = BuildParameters.Paths.Directories.PublishedLibraries.Combine(parsedProject.RootNameSpace);
+                EnsureDirectoryExists(outputFolder);
+                Information(parsedProject.OutputPath.FullPath);
+                CopyFiles(GetFiles(parsedProject.OutputPath.FullPath + "/**/*"), outputFolder, true);
+            }
             continue;
         }
     }

